src/connector/src/parser/unified/debezium.rs

```diff
@@ -164,18 +162,34 @@ pub struct MongoProjection<A> {
     accessor: A,
 }
 
-pub fn extract_bson_id(id_type: &DataType, bson_doc: &serde_json::Value) -> anyhow::Result<Datum> {
-    let id_field = bson_doc
-        .get("_id")
-        .ok_or_else(|| anyhow::format_err!("Debezuim Mongo requires document has a `_id` field"))?;
+pub fn extract_bson_id(id_type: &DataType, bson_doc: &serde_json::Value) -> AccessResult {
+    let id_field = bson_doc.get("_id").ok_or_else(|| AccessError::Undefined {
+        name: "_id".to_owned(),
+        path: "Debezuim Mongo".to_owned(),
```

Comment:
I think the `path` here is not correct. Is it just a placeholder?

Reply:
I didn't find a better value for this. 😕 Maybe we can use an uncategorized variant.